src/components/Player.js

```diff
@@ -15,8 +16,18 @@ export default function Player(props) {
     var internalPlayer = props.playerRef.current?.getInternalPlayer()
     var videoUrl = internalPlayer.getVideoUrl()
     if (!videoUrl) return
-    console.log(`BROKEN VIDEO: ${getVideoId(videoUrl)}`)
-    internalPlayer.nextVideo()
+
+    var videoId = getVideoId(videoUrl)
+    console.log(`BROKEN VIDEO: ${videoId}`)
+    Honeybadger.notify(`BROKEN VIDEO: ${videoId}`);
```

Comment:
Should we add anything else to the context? Playlists maybe?

Reply:
I could give the current videos list unless there is some limit on the value, 

OR just the video of the player's current index

Reply:
Added player's current video